src/api/api.ts

```diff
@@ -0,0 +1,29 @@
+import { client } from '../utils/fetchClient';
+import { User } from '../types/User';
+import { Post } from '../types/Post';
+import { Comment } from '../types/Comment';
+
+export const getUser = () => {
+  return client.get<User[]>('/users');
+};
+
+export const getPosts = ({ id }: Pick<User, 'id'>) => {
+  return client.get<Post[]>(`/posts?userId=${id}`);
+};
+
+export const getComments = (postId: number) => {
+  return client.get<Comment[]>(`/comments?postId=${postId}`);
+};
+
+export const deleteComment = (commentId: number) => {
+  return client.delete(`/comments/${commentId}`);
+};
```

Comment:
The `deleteComment` function should include error handling to manage potential errors during the deletion process. Consider adding a `.catch()` block to handle any errors that may occur.